tests/Testcontainers.ArangoDb.Tests/ArangoDbContainerTest.cs

```diff
@@ -26,7 +26,7 @@ public async Task RetrievesDatabases()
         using var client = new ArangoDBClient(transport);
 
         // When
-        var response = await client.Database.GetDatabasesAsync()
+        var response = await client.Database.GetDatabasesAsync(TestContext.Current.CancellationToken)
```

Comment:
Does it make sense to pass the cancellation token to every call if we're never actually canceling the tests? Is there any benefit to doing that? (I haven't looked into the v3 features and changes yet).

Reply:
While it fixes https://xunit.net/xunit.analyzers/rules/xUnit1051 it probably is only of use, when you actually cancel a run or set a timeout for a test. As you cancel all running jobs after the first test-failure, it should be beneficial in the pipeline, and probably even more so locally, when you realise that you hit "run tests" just that moment to early. If you object, I can remove the token and suppress the analyser.

Reply:
Thanks for sharing. I'll try to finish the review by the end of this week.